frontend/package.json

```diff
@@ -29,10 +32,17 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "sirv": "^2.0.3",
+    "styled-components": "^6.1.8",
     "ts-node": "^10.9.1",
     "typescript": "^5.3.3",
     "vike": "^0.4.156",
     "vite": "^5.0.10"
   },
-  "type": "module"
```

Comment:
This line got dropped for some reason which results in the following when you run build / test:

```
The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
```